src/main/webapp/issueList/IssueList.js

```diff
@@ -0,0 +1,222 @@
+
+const stylesHeader = {
+  pageTitle: {
+    fontWeight: '700', margin: '20px 0', fontSize: '35px', display: 'block'
+  }
+};
+
+const SPINNER_NAME = 'ISSUE_LIST';
+const styles = {
+  project: {
+    projectKeyWidth: '140px',
+    projectWidth: '750px',
+    titleWidth: '280px',
+    typeWidth: '193px'
+  }
+}
+
+const SIZE_PER_PAGE_LIST = [
+  { text: '10', value: 10 },
+  { text: '25', value: 25 },
+  { text: '50', value: 50 },
+  { text: '100', value: 100 }];
+
+const columns = [
+  {
+    dataField: 'id',
+    text: 'Id',
+    hidden: true,
+    csvExport: false
+  }, {
+    dataField: 'projectKey',
+    text: 'Project',
+    sort: true,
+    headerStyle: (column, colIndex) => {
+      return { width: styles.project.projectKeyWidth };
+    },
+    formatter: (cell, row, rowIndex, colIndex) =>
+      div({}, [
+        h(Link, { to: { pathname: '/project/main', search: '?projectKey=' + row.projectKey, state: { issueType: 'project', projectKey: row.projectKey } } }, [row.projectKey])
+      ])
+  }, {
+    dataField: 'summary',
+    text: 'Title',
+    sort: true,
+    headerStyle: (column, colIndex) => {
+      return { width: styles.project.titleWidth };
+    },
+    formatter: (cell, row, rowIndex, colIndex) =>
+      div({}, [
+        h(Link, { to: { pathname: '/project/main', search: '?projectKey=' + row.projectKey, state: { issueType: 'project', projectKey: row.projectKey } } }, [row.summary])
+      ])
+  }, {
+    dataField: 'type',
+    text: 'Type',
+    sort: true,
+    headerStyle: (column, colIndex) => {
+      return { width: styles.project.typeWidth };
+    }
+  }, {
+    dataField: 'status',
+    text: 'Status',
+    sort: false
+  }, {
+    dataField: 'updateDate',
+    text: 'Updated',
+    sort: false,
+    classes: 'ellipsis-column'
+  },
+  {
+    dataField: 'actors',
+    text: 'Awaiting action from',
+    sort: true,
+    formatter: (cell, row, rowIndex, colIndex) =>
+      div({}, [
+        !isNil(row.actors) ? row.actors.join(", ") : ''
+      ]),
+    csvFormatter: (cell, row, rowIndex, colIndex) =>
+      !isNil(row.actors) ? row.actors.join(", ") : ''
+  }
+];
+
+export const IssueList = hh(class IssueList extends Component {
+  
+  paramsContext = new URLSearchParams();
+  _isMounted = false;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      sizePerPage: 10,
+      search: null,
+      sort: {
+        sortDirection: 'asc',
+        orderColumn: null
+      },
+      currentPage: 1,
+      issues: []
+    };
+  }
+
+  componentDidMount() {
+    this._isMounted = true;
+    this.paramsContext = new URLSearchParams(this.props.location.search);
+    this.init();
+  }
+
+  componentWillUnmount() {
+    this._isMounted = false;
+  }
+
+  init = () => {
+    spinnerService.show(SPINNER_NAME);
+    this.tableHandler(0, this.state.sizePerPage, this.state.search, this.state.sort, this.state.currentPage);
+  };
+
+  tableHandler = (offset, limit, search, sort, page) => {
+    if(this._isMounted) {
```

Comment:
`if (this._isMounted) { }` should be after the resolve of the promise is resolved